ci/run-ci.py

```diff
@@ -137,8 +137,11 @@ def arg_parse():
     if args['no_cache']:
         KANIKO_CACHE_TTL = '-1h'
 
+    print(f'💃💃💃\n')
```

Comment:
I think there's no need for the `\n ` here, there will be a line return automatically.

Reply:
Thanks! It turns out the newline looks good in this context, so I'll just leave it.